cordova-lib/src/cordova/plugin.js

```diff
@@ -298,8 +326,24 @@ module.exports = function plugin(command, targets, opts) {
                                     configXml.write();
                                 }
                             }
-                        })
-                        .then(function(){
```

Comment:
Not sure why this was deleted? Is that okay? I retested everything and it worked....